src/utils.js

```diff
@@ -25,4 +25,33 @@ const parseBearerToken = (rawAuthorizationHeader) => {
   return split[1];
 };
 
-module.exports = { generateSignedJWT, parseBearerToken };
+/**
+ * Get date from 
+ * @param {string} rawAuthorizationHeader
+ * @returns
+ */
+ const getDate = (dateObj, orderBy) => {
```

Comment:
This variable name could also be improved to better reflect what exactly this function is doing.